speculos/main.py

```diff
@@ -307,6 +307,13 @@ def main(prog=None):
         logger.error(f"Invalid api_level {args.apiLevel}")
         sys.exit(1)
 
+    # Set SPECULOS_DETECTED_APPNAME env variable for proper emulation.
+    # See sys_os_registry_get_current_app_tag() for corresponding usage.
+    app_name = metadata.get("app_name")
+    app_version = metadata.get("app_version")
+    if app_name and app_version:
```

Comment:
No else ?

Reply:
When `app_name` and `app_version` are not available, the default value will be set by the behavior of `sys_os_registry_get_current_app_tag()`. so IMO there is nothing else to do?

Reply:
Ok